classifier_and_summarizer/classifier/classifier.py

```diff
@@ -0,0 +1,30 @@
+class Classifier:
+    '''Class to determine if a webpage is stampable
+
+    Detemines if a webpage is stampable based on the following
+    max(media_count,text_count) + embedded_content_count >= min_pages
+    '''
+
+    def __init__(
+            self,
+            normal_text_contents,
+            title_text_contents,
+            media_contents,
+            embedded_contents,
+            max_pages):
+        self.normal_text_count = len(normal_text_contents)
+        self.title_text_count = len(title_text_contents)
+        self.media_count = len(media_contents)
+        self.embedded_content_count = len(embedded_contents)
+        self.min_pages = max_pages // 2
```

Comment:
What is max_pages here?

Reply:
max_pages is the maximum number of stamp pages allowed in the final stamp of the website - this is a value supplied by the user. I've assumed we need at least max_pages//2 for the website to be stampifiable.

Reply:
Makes sense! Add a comment for the same here.